### PR TITLE
HEC- 784: license expiry date

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: test/uk/gov/hmrc/hecapplicantfrontend/models/UserAnswersSpec.scala [hunk 1]
```diff
@@ -59,8 +66,11 @@ class UserAnswersSpec extends AnyWordSpec with Matchers {
 
       "unsets the licence type field" in {
```
Comment (priyankadhiman3006) [inline test/uk/gov/hmrc/hecapplicantfrontend/models/UserAnswersSpec.scala:67]: Should we say unsets the licence type and expiry date field?

**Review by priyankadhiman3006 — APPROVED** [review]

File: test/uk/gov/hmrc/hecapplicantfrontend/models/UserAnswersSpec.scala (hunk 1 above)
Comment (AndyHWChung) [reply test/uk/gov/hmrc/hecapplicantfrontend/models/UserAnswersSpec.scala:67]: oh yeah - my intention here was to have separate parts of the specs to cover the different fields, so expiry date unstting would be done in a separate `in` block

Comment (AndyHWChung) [reply test/uk/gov/hmrc/hecapplicantfrontend/models/UserAnswersSpec.scala:67]: that way we have tests which are generally testing one thing at a time

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**